contributors.md

```diff
@@ -0,0 +1,27 @@
+# Contributing to Docs Authoring Pack 
+
+Interested in making a code contribution and not sure where to begin? Check out our help wanted issues. If you want to make an enhancement, create an issue with the proposed change for review. 
+
+## Pull Requests 
+
+When you are ready to submit your contribution, review this checklist before submitting your PR: 
+
+[ ] Verify all tests pass by running: `npm run test` 
```

Comment:
Hey Adam, it looks like the checklist runs together if I [view the file](https://github.com/microsoft/vscode-docs-authoring/blob/b5d767e5ab7ea535b76dc1535b78c31246874da3/contributors.md) or check out preview locally so we may need to add a blank line but can you confirm you experience the same behavior?  Thanks

Reply:
Good catch!

Reply:
Does it look better @jamarw ?

Reply:
@adunndevster , looks great!